packages/cli/src/Doctor.ts

```diff
@@ -64,6 +65,8 @@ ${chalk.bold('Examples')}
       return this.help(args.message)
     }
 
+    await checkUnsupportedDataProxy('doctor', args, true)
```

Comment:
`doctor` => command name
`args` => command flags
`true` => implicitly load `schema.prisma`

Reply:
Implicit refers to the behavior of some of the cli commands. Some of them, will load the schema at it's default location, and some others won't, and some others will only do it conditionally. I did not want to alter or touch this logic, so it is reflected by a boolean instead.